src/app/main/component/ubs/services/order.service.ts

```diff
@@ -17,8 +17,11 @@ export class OrderService {
   locationSubject = new Subject();
   locationSub = new Subject();
   currentAddress = new Subject();
+  getStepperFinalSubject = new BehaviorSubject<boolean>(false);
```

Comment:
It's property but not a method so you shouldn't name it like a method
It should be something like `isStepperOnFinalStep`
Because with current name it's hard to understand what type of information this subject contains

Reply:
done local